### PR TITLE
 Add a dummy test to enable CI

### PR DESCRIPTION
**Pull request by sthaha**
Add a dummy test to enable CI

    This patch
     - uses `tektoncd/plumbing` to setup various jobs
     - adds a dummy go test
     - starts a postgres container that is needed by API tests

    Fixes: https://github.com/tektoncd/hub/issues/15

Signed-off-by: Sunil Thaha <sthaha@redhat.com>

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# to run a specific set of tests.
+
+# Markdown linting failures don't show up properly in Gubernator resulting
+# in a net-negative contributor experience.
+export DISABLE_MD_LINTING=1
+
+source $(dirname $0)/../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
+
+echo "CI works fine"
+exit 1
```
Comment (vdemeester) [inline test/presubmit-tests.sh:31]: This will make the CI red though :angel:

Comment (sthaha) [reply test/presubmit-tests.sh:31]: yes, on purpose  :D 
1.  I wanted to show how the CI is working 
2. I want to add a dummy_test.go to api and run that before making the CI green 👼

**Comment by sthaha** [comment]
@vdemeester is there a way to generate coverage info for just the `api`  directory (go code) ?

File: test/presubmit-tests.sh (hunk 1 above)
Comment (sthaha) [reply test/presubmit-tests.sh]: Resolving it since we have the go test (dummy) added.

**Comment by vdemeester** [comment]
> @vdemeester is there a way to generate coverage info for just the `api` directory (go code) ?

there might be yes :upside_down_face:

File: test/presubmit-tests.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+api-test(){
+  (
+    cd $SCRIPT_DIR/../api
+    api-unittest
+  )
+}
+
+main() {
```
Comment (vdemeester) [inline test/presubmit-tests.sh:47]: Any reason for not using the `presubmit` main ?

Comment (sthaha) [reply test/presubmit-tests.sh:47]: good question .. didn't even cross my mind 🤦‍♂️

**Pull request merged**